fake_switches/cisco/command_processor/config_interface.py

```diff
@@ -166,7 +168,8 @@ def do_no_ip(self, *args):
                 self.port.vrf = None
         if "redirects".startswith(args[0]):
             self.port.ip_redirect = False
-
+        if "verify unicast".startswith(args[0]) and isinstance(self.port, VlanPort):
```

Comment:
I'm pretty sure that we split on space and that arg[0] will be "verify" and arg[1] will be "unicast".  So "verify multicast" for example will still fall in this if statement.